data/Music/Mario & Luigi Series/Mario & Luigi Superstar Saga/Come On!-ARG.meta

```diff
@@ -0,0 +1,4 @@
+Mario & Luigi Superstar Saga - Come On!-ARG
```

Comment:
For consistency, would be cool to rename the title like this:
Mario & Luigi Superstar Saga - Come On! V2